docs/success_metrics.md

```diff
@@ -0,0 +1,23 @@
+## Success Metrics
+
+DPUL & DPUL Collections is a success if patrons are inspired towards transformative use of PUL Material, whether they knew we had that material or not.
```

Comment:
suggestion: DPUL & DPUL Collections are a success if patrons are inspired transformatively use PUL material, whether they knew we had that material or not.